src/main/java/com/base/BaseScheduledFlow.java

```diff
@@ -0,0 +1,54 @@
+package com.base;
+
+import com.base.processors.ContactProcessor;
+import com.base.processors.DealProcessor;
+import com.getbase.models.*;
+import com.getbase.sync.Sync;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import static com.base.util.BaseAppUtil.baseClient;
+import static com.base.util.BaseAppUtil.getProperty;
+
+/**
+ * Created by przemek on 19.10.2016.
+ */
+
+@Service
+@Slf4j
+class BaseScheduledFlow {
+
+    @Autowired
+    private DealProcessor dealProcessor;
+
+    @Autowired
+    private ContactProcessor contactProcessor;
+
+    public BaseScheduledFlow() {
+        log.info("BaseScheduledFlow service launching...");
+    }
+
+
+    @Scheduled(fixedDelay = 15000)
+    public void process() {
+        Sync sync = new Sync(baseClient(), getProperty("deviceUUID"));
```

Comment:
Spring can inject property without `getProperty` explicit method call (`@Value` in `@Autowired` constructor)